.github/workflows/distribute-apps.yml

```diff
@@ -54,7 +68,7 @@ jobs:
 
       - name: Retrieve the secret and decode it to a file
         env:
-          ADMIN_SDK_BASE64: ${{ secrets.MURRAY_APPS_DEV_FIREBASE_ADMIN_BASE_64_ENCODED_JSON }}
+          ADMIN_SDK_BASE64: ${{ secrets.FIREBASE_ADMIN_BASE_64_ENCODED_JSON }}
```

Comment:
Have the secrets been renamed and then put into environments?

Reply:
Looks like it's there 
<img width="445" alt="image" src="https://github.com/user-attachments/assets/b048447e-f182-4276-a42a-2307be43bdcb">